src/react/hooks/use-messages.ts

```diff
@@ -37,36 +37,151 @@ import { useChatConnection } from './use-chat-connection.js';
 export interface UseMessagesResponse extends ChatStatusResponse {
   /**
    * A shortcut to the {@link Messages.send} method.
+   *
+   * This is a stable reference and will not be changed between renders for the same room.
+   * @example
+   * ```tsx
+   * const { sendMessage } = useMessages();
+   *
+   * const handleSend = async () => {
+   *   try {
+   *     await sendMessage({
+   *       text: 'Hello world!',
+   *     });
+   *   } catch (error) {
+   *     console.error('Failed to send message:', error);
+   *   }
+   * };
+   * ```
    */
   readonly sendMessage: Messages['send'];
 
   /**
    * A shortcut to the {@link Messages.get} method.
+   *
+   * This is a stable reference and will not be changed between renders for the same room.
+   * @example
+   * ```tsx
+   * const { getMessage } = useMessages();
+   *
+   * const fetchSpecificMessage = async (messageSerial: string) => {
+   *   try {
+   *     const message = await getMessage(messageSerial);
+   *     console.log('Retrieved message:', message.text);
+   *   } catch (error) {
+   *     console.error('Failed to get message:', error);
+   *   }
+   * };
+   * ```
    */
   readonly getMessage: Messages['get'];
 
   /**
    * A shortcut to the {@link Messages.update} method.
```

Comment:
I've been thinking about this, would it be better to actually describe the method in full rather than just linking? How much can we rely on humans/machines actually following the trail?